app/src/main/java/org/wikipedia/page/edithistory/EditHistoryListActivity.kt

```diff
@@ -122,6 +162,7 @@ class EditHistoryListActivity : BaseActivity() {
 
         private const val VIEW_TYPE_SEPARATOR = 0
         private const val VIEW_TYPE_ITEM = 1
+        private const val VIEW_TYPE_STATS = 2
```

Comment:
On second thought, instead of a `NestedScrollView`, we could use a separate Adapter that holds the Stats view, and `concat` the adapter onto the compound adapter that I'm already using with the header/footer views. We can discuss further in the next sync.

Reply:
Got it, I think putting the stats view outside the `RecyclerView` will be easier to do and I'll do that.